package.nls.json

```
@@ -183,6 +183,7 @@
     "AWS.command.quickStart.error": "There was an error retrieving the Quick Start page",
     "AWS.command.s3.downloadFileAs": "Download As...",
     "AWS.command.s3.copyPath": "Copy Path",
+    "AWS.command.s3.presignedURL": "Generate Presigned URL...",
```

"..." implies extra steps, but I think this is a one-step command?
```suggestion
    "AWS.command.s3.presignedURL": "Copy Presigned URL",
```